packages/nyc-test-coverage/src/index.ts

```diff
@@ -23,7 +25,7 @@ export class WorkflowCoverage {
       throw new TypeError('Cannot automatically instrument coverage without specifying `workflowsPath`');
     }
 
-    if (!hasCoverageGlobal) {
+    if (!this.hasCoverageGlobal()) {
```

Comment:
Why is this necessary?

Reply:
This avoids instrumenting Workflow code if coverage isn't enabled. I think it is beneficial to avoid piping the Workflow code through istanbul-instrumenter-loader if coverage isn't enabled. I don't expect it to cause any problems, but I'd prefer to err on the side of caution there.

Reply:
I'd throw an error here instead then instead of silently failing.

Reply:
Do you want to put the responsibility on the user to avoid calling this function if code coverage is disabled?

Reply:
Do you expect this to be a common scenario?

Reply:
I think this is very common, IIUC. The user doesn't want to have to rip out their workflowCoverage augmentation code just because they disabled coverage on a specific test run.

Reply:
SGTM, please ignore this comment then.
I would maybe consider issuing a warning to let the user know that coverage is disabled.

Reply:
I agree with @joebowbeer . It isn't uncommon to run tests with coverage on vs coverage off. And that means every user would have to implement their own check for whether they're running with coverage or not.